HotelBooking/Views/cancel_reservation_view.py

```diff
@@ -116,4 +114,3 @@ def initiate_options(self):
             }
             choices.append(choice)
         PROMPTS[PROMPT_KEY['OPERATIONS']][0]['choices'] = choices
-        
```

Comment:
Remove your changes for this file, it is not a model and will conflict with view comment changes.

Reply:
fixed